test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/command/http/HttpCommand.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Internal;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.entity;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Polly;
+using Polly.Retry;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.command.http
+{
+    public class HttpCommand : Command<HttpResponseMessage>
+    {
+        private HttpRequestEntity httpRequestEntity;
+        private HttpClient httpClient;
+        
+        private AsyncRetryPolicy retryPolicy = Policy.Handle<HttpRequestException>()
+            .WaitAndRetryAsync(
+               retryCount: 6,
+               sleepDurationProvider: _ => TimeSpan.FromSeconds(20)
+            );
+               
+        private HttpCommand(HttpCommandBuilder httpCommandBuilder)
+        {
+            this.httpRequestEntity = httpCommandBuilder.GetHttpRequestEntity();
+            httpClient = new HttpClient();
+        }
+
+        public void Dispose()
+        {
+            throw new NotImplementedException();
+        }
+
+
+        public async Task<HttpResponseMessage> ExecuteCommandAsync()
+        {
+            string httpMethod = httpRequestEntity.GetHttpMethod();
+
+            if(httpMethod.Equals(HttpMethods.Post))
```

Comment:
Requesting you to refactor this to switch case scenario

Reply:
This is based on non-constant string comparison and therefore in if else condition. Creating constants for switch case.

Reply:
Done.